fhd_core/beam_modeling/fhd_struct_init_psf.pro

```diff
@@ -9,13 +9,13 @@ FUNCTION fhd_struct_init_psf,beam_ptr=beam_ptr,complex_flag=complex_flag,$
     n_pol=n_pol,n_freq=n_freq,freq_cen=freq_cen,pol_norm=pol_norm,freq_norm=freq_norm,group_arr=group_arr,$
     interpolate_kernel=interpolate_kernel,beam_mask_threshold=beam_mask_threshold,$
     image_power_beam_arr=image_power_beam_arr,ra_arr=ra_arr,dec_arr=dec_arr,$
-    psf_image_resolution=psf_image_resolution,psf_image_dim=psf_image_dim   
- 
+    psf_image_resolution=psf_image_resolution,psf_image_dim=psf_image_dim
+
 IF N_Elements(n_pol) EQ 0 THEN n_pol=1 ELSE n_pol=Fix(n_pol)
 IF N_Elements(n_freq) EQ 0 THEN n_freq=1 ELSE n_freq=Fix(n_freq)
 IF N_Elements(freq_cen) EQ 0 THEN freq_cen=Fltarr(n_freq) ELSE freq_cen=Float(freq_cen)
 IF N_Elements(beam_ptr) EQ 0 THEN beam_ptr=Ptr_new() ;will actually have dimensions (npol,nfreq,nbaselines)
-IF N_Elements(psf_resolution) EQ 0 THEN psf_resolution=1 ELSE psf_resolution=Fix(psf_resolution);over-resolution
+IF N_Elements(psf_resolution) EQ 0 THEN psf_resolution=16. ELSE psf_resolution=Fix(psf_resolution);over-resolution
```

Comment:
I would actually advocate changing this to simply:
`psf_resolution=Fix(psf_resolution);over-resolution`
None of the other 'defaults' in this file are real default values, and a proper cleanup would remove the appearance of defaults and let things crash here if they are not defined previously.

Reply:
Do you mean to remove the if/then/else statement entirely, and just have the result of the else?

Reply:
That seems the best to me now, since it will raise the most helpful error message and prevent the code from running with values that might not be what the user wants (typos happen).